tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py

```diff
@@ -2158,7 +2169,11 @@ def test_OnDiskDataset_load_feature(edge_fmt):
             f.write(yaml_content)
 
         # Case1. Test modify the `in_memory` field.
-        dataset = gb.OnDiskDataset(test_dir).load()
+        with pytest.warns(
+            DGLWarning,
+            match="Edge feature is stored, but edge IDs are not saved.",
+        ):
+            dataset = gb.OnDiskDataset(test_dir).load()
```

Comment:
why not adding `include_original_edge_id=True`? please note there are 2 calls of `gb.OnDiskDataset()` in this testcase.

Reply:
In fact, this test contains not 2, but 5 calls of `gb.OnDiskDataset()`, and one way to resolve these warnings is to use `include_original_edge_id=True` for all of them. The other way would be the use 
```
        with pytest.warns(
            DGLWarning,
            match="Edge feature is stored, but edge IDs are not saved.",
        ):
            dataset = gb.OnDiskDataset(test_dir).load()
```
only on the first such call.

We have similar situations in
```
# of calls               TestName:
    5                test_OnDiskDataset_load_feature                                       - 
    5                test_OnDiskDataset_load_graph                                    
    6                test_OnDiskDataset_load_tasks
    6                test_OnDiskDataset_load_tasks_selectively
```
Do you want me to use `include_original_edge_id=True` everywhere?

Reply:
use include_original_edge_id=True everywhere

Reply:
DONE